recipes/reacnetgenerator/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "reacnetgenerator" %}
+{% set version = "1.4.89" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmodeling/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: fe91926c3885d6bb12b43d16012a956102165f9e72a03b060c301f551800b240
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  entry_points:
+    - reacnetgenerator = reacnetgenerator.commandline:_commandline
+    - reacnetgeneratorgui = reacnetgenerator.gui:gui
+  script: "SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - yarn
+  host:
+    - python
+    - pip
+    - cython >=0.24.1
+    - setuptools_scm
+    - pytest-runner
+    - numpy >=1.15
+  run:
+    - python
+    - openbabel
+    - rdkit >=2018.09
+    - {{ pin_compatible('numpy') }}
+    - scipy >=0.20.1
+    - pandas
+    - networkx
+    - matplotlib
+    - hmmlearn >=0.2.1
+    - ase
+    - scour
+    - tqdm
+    - coloredlogs
+    - lz4
+    - pybase64
+    - requests
+
+test:
+  imports:
+    - reacnetgenerator
+  commands:
+    - reacnetgenerator -h
+
+about:
+  home: http://github.com/tongzhugroup/reacnetgenerator
+  license: LGPL-3.0
+  license_family: LGPL
+  license_file: LICENSE
+  summary: 'An automatic generator of reaction network for reactive molecular dynamics simulation.'
+  doc_url: https://reacnetgenerator.njzjz.win/
+  dev_url: https://github.com/tongzhugroup/reacnetgenerator
+
+extra:
+  recipe-maintainers:
+    - njzjz
+    - DiracMD
```

Comment:
@DiracMD please confirme you are willing to be listed here.

Reply:
yes?is it ok ?